wolfcrypt/test/test.c

```diff
@@ -30215,7 +30251,7 @@ static int myDecryptionFunc(PKCS7* pkcs7, int encryptOID, byte* iv, int ivSz,
             keyId = 1;
         }
 #else
-        keyId = *(int*)(keyIdRaw + 2);
+        XMEMCPY(&keyId, keyIdRaw + 2, sizeof keyId);
```

Comment:
Nice cleanup to ensure no unaligned issues. Please use `sizeof(keyId)`. Thanks

Reply:
done